core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -732,27 +745,69 @@ portfolio.pending.tradePeriodInfo=After the first blockchain confirmation, the t
+
+portfolio.pending.support.headline.getHelp=Need help?
+portfolio.pending.support.text.getHelp=If you have any problems you can try to contact the trade peer in the trade \
+  chat or ask the Bisq community at https://bisq.community. \
+  If you cannot resolve your problems you can request help from the mediator.
+portfolio.pending.support.button.getHelp=Get support
+portfolio.pending.support.popup.info=If you cannot resolve your problems with the trade peer you can open a support \
+  ticket to request help from the mediator. If you have not received the payment wait until the trade period is over.\n\n\
+  Are you sure you want to open a support ticket?
+portfolio.pending.support.popup.button=Open support ticket
+portfolio.pending.support.headline.halfPeriodOver=Check payment
+portfolio.pending.support.headline.periodOver=Trade period is over
+
+portfolio.pending.arbitrationRequested=Arbitration requested
+portfolio.pending.mediationRequested=Mediation requested
 portfolio.pending.openSupport=Open support ticket
 portfolio.pending.supportTicketOpened=Support ticket opened
 portfolio.pending.requestSupport=Request support
-portfolio.pending.error.requestSupport=Please report the problem to your arbitrator.\n\nHe will forward the information to the developers to investigate the problem.\nAfter the problem has been analyzed you will get back all locked funds.
+portfolio.pending.error.requestSupport=Please report the problem to your mediator or arbitrator.\n\nHe will forward the \
+  information to the developers to investigate the problem.\nAfter the problem has been analyzed you will \
+  get back all locked funds.
 portfolio.pending.communicateWithArbitrator=Please communicate in the \"Support\" screen with the arbitrator.
+portfolio.pending.communicateWithMediator=Please communicate in the \"Support\" screen with the mediator.
 portfolio.pending.supportTicketOpenedMyUser=You opened already a support ticket.\n{0}
 portfolio.pending.disputeOpenedMyUser=You opened already a dispute.\n{0}
 portfolio.pending.disputeOpenedByPeer=Your trading peer opened a dispute\n{0}
 portfolio.pending.supportTicketOpenedByPeer=Your trading peer opened a support ticket.\n{0}
 portfolio.pending.noReceiverAddressDefined=No receiver address defined
-portfolio.pending.removeFailedTrade=If the arbitrator could not close that trade you can move it yourself to the failed trades screen.\n\
+portfolio.pending.removeFailedTrade=If the mediator or arbitrator could not close that trade you can move it yourself \
+  to the failed trades screen.\n\
   Do you want to remove that failed trade from the Pending trades screen?
+
+portfolio.pending.mediationResult.headline=Suggested payout from mediation
+portfolio.pending.mediationResult.info.noneAccepted=Complete the trade by accepting the mediator's suggestion for the trade payout.
+portfolio.pending.mediationResult.info.selfAccepted=You have accepting the mediator's suggestion. Waiting for peer to accept as well.
```

Comment:
@chimp1984 Great work as always! 👍 Before I squash and merge the commits (just too many commits to overlook everything) I'd like to have a text review by @m52go before. @m52go Could you please go over the new translations for the mediation system? e.g. here I think it should be `You have accepted the mediator's sugestion....`